contrib/kubespray/config/config.yaml

```diff
@@ -0,0 +1,84 @@
+user: forexample
+password: forexample
+branch_name: <% latest-release %>
```

Comment:
Please use something like `pai-1.4.y`

Reply:
> Please use something like `pai-1.5.y`

I'll create a wiki for the release manager on how to publish release & leave it `pai-1.4.y` for instance.

Reply:
Maybe `pai-1.5.y`? Because 1.4.0 is released and cannot use this installation script.

Reply:
> Maybe `pai-1.5.y`? Because 1.4.0 is released and cannot use this installation script.

Changed, thx